app/controllers/submissions_controller.rb

```diff
@@ -243,21 +243,28 @@ def download
           problem_name = "General"
           if !problem.nil? then problem_name = problem.name end
 
-          comment = "#{annotation.comment}\n\nProblem: #{problem_name}\nScore:#{value}"
+          comment = "#{annotation.comment}\n\nProblem: #{problem_name}\nScore: #{value}"
 
           # + 1 since pages are indexed 1-based
           pdf.go_to_page(page + 1)
-
+          
+          # Below is old code for drawing boxes and adding text to pdf
```

Comment:
I think we can remove this chunk of old code, doubt anyone will miss it. 
I have also fixed the header_position checking logic with the change in https://github.com/autolab/Autolab/pull/1068
Otherwise the backend code is LGTM - I will defer the frontend to @jbman223 !